src/routes/channels/+page.svelte

```diff
@@ -0,0 +1,114 @@
+  import SummaryRow from '$lib/elements/SummaryRow.svelte'
+
+  $: sendableMsat =
+    $channels$.data &&
+    $channels$.data.reduce(
+      (acc, { balanceLocal, reserveLocal }) =>
+        Big(acc).add(balanceLocal).minus(reserveLocal).toString(),
+      '0'
+    )
+
+  $: receivableMsat =
+    $channels$.data &&
+    $channels$.data.reduce(
+      (acc, { balanceRemote, reserveRemote }) =>
+        Big(acc).add(balanceRemote).minus(reserveRemote).toString(),
+      '0'
+    )
+</script>
+
+<svelte:head>
+  <title>{$translate('app.titles./channels')}</title>
+</svelte:head>
+
+<section class="flex flex-col justify-center items-start w-full p-4 max-w-lg">
+  <div class="flex items-center justify-between mb-6 mt-12 w-full">
+    <div class="flex items-center">
+      <div class="w-10 mr-2">{@html channels}</div>
+      <h1 class="text-4xl font-bold">
+        {$translate('app.titles./channels')}
+      </h1>
+    </div>
+
+    <div>
+      <Button on:click={() => goto('/channels/open')} text={$translate('app.buttons.open')} primary>
+        <div slot="iconLeft" class="w-8 mb-[1px] -ml-2">
+          {@html plus}
+        </div>
+      </Button>
+    </div>
+  </div>
+
+  <div class="w-full flex justify-center">
+    {#if $channels$.loading}
+      <Spinner />
+    {:else if $channels$.error}
+      <ErrorMsg message={$channels$.error} />
+    {:else if $channels$.data}
+      <div class="w-full">
+        <div class="w-full mb-6">
+          <h2 class="font-bold text-lg underline underline-offset-2 mb-2">Summary</h2>
+
+          <SummaryRow>
+            <div slot="label">Channels:</div>
+            <div slot="value">{$channels$.data.length}</div>
+          </SummaryRow>
+
+          <SummaryRow>
+            <div slot="label">{$translate('app.labels.sendable')}:</div>
+            <div slot="value">
+              {formatValueForDisplay({
+                value: convertValue({
+                  value: sendableMsat,
+                  from: BitcoinDenomination.msats,
+                  to: $settings$.primaryDenomination
+                }),
+                denomination: $settings$.primaryDenomination,
+                commas: true
+              })}
+            </div>
+          </SummaryRow>
+
+          <SummaryRow>
+            <div slot="label">{$translate('app.labels.receivable')}:</div>
+            <div slot="value">
+              {formatValueForDisplay({
+                value: convertValue({
+                  value: receivableMsat,
+                  from: BitcoinDenomination.msats,
+                  to: $settings$.primaryDenomination
+                }),
+                denomination: $settings$.primaryDenomination,
+                commas: true
+              })}
+            </div>
+          </SummaryRow>
+        </div>
+
+        <div class="w-full">
+          <h2 class="font-bold text-lg underline underline-offset-2 mb-2">Open channels</h2>
+
+          <div class="w-full flex-grow overflow-y-auto gap-y-4">
```

Comment:
```suggestion
          <div class="w-full h-64 flex-grow overflow-y-auto gap-y-4">
```

With large amount of channels, UI is broken. Overflow isn't working unless there is a fixed height I think? 

![Screen Shot 2023-07-07 at 12 52 06 PM](https://github.com/clams-tech/browser-app/assets/30157175/b2e54e6a-a2c8-49b5-8c4e-60d8ae8d7e3d)

With fixed height applied: 
![Screen Shot 2023-07-07 at 12 54 59 PM](https://github.com/clams-tech/browser-app/assets/30157175/57e75303-5dc7-485c-a919-0b464b7da5a3)

Reply:
Fixed!